features/booking/extract_data.feature

```diff
@@ -17,6 +17,26 @@ Feature: Extract booking data
   Scenario: Isolating the changed data to allow selective processing
     When the booking data is a mix of new and already extracted data
     Then only the changed booking data is extracted
+  
+  @wip
```

Comment:
This is outside the scope of this pull request as I am trying to focus on data extraction only. 

I plan to do a review of the features as I implement them, and then do a full review with @amansinghb once all the code bits I have been working on have been merged.